modules/publication/jsx/projectFields.js

```diff
@@ -0,0 +1,378 @@
+              options={this.props.uploadTypes}
+              required={true}
+            />
+            <TextboxElement
+              name={publicationCitation}
+              label="Citation"
+              onUserInput={this.props.setFormData}
+              value={this.props.formData[publicationCitation]}
+            />
+            <TextboxElement
+              name={publicationVersion}
+              label="Publication Version"
+              onUserInput={this.props.setFormData}
+              value={this.props.formData[publicationVersion]}
+            />
+          </div>
+        );
+      }
+    }
+
+    return fileFields;
+  }
+
+  createCollabEmailFields() {
+    let collabEmails = [];
+    if (this.props.formData.collaborators) {
+      this.props.formData.collaborators.forEach(
+        function(c, i) {
+          let name = 'collabEmail_' + c.name;
+          collabEmails.push(
+            <EmailElement
+              name={name}
+              label={c.name + (c.name.slice(-1) === 's' ? "'" : "'s") + " Email"}
+              onUserInput={this.setCollaboratorEmail}
+              onUserBlur={this.props.validateEmail}
+              toggleEmailNotify={this.toggleEmailNotify}
+              errorMessage={this.props.formErrors[name]}
+              required={false}
+              value={this.props.formData.collaborators[i].email}
+              addressee={c.name}
+            />
+          );
+        }, this);
+    }
+    return collabEmails;
+  }
+
+  addCollaborator(formElement, value, pendingValKey) {
+    let collaborators = this.props.formData.collaborators || [];
+    collaborators.push(
+      {
+        name: value,
+        email: null,
+        notify: false
+      }
+    );
+
+    this.props.setFormData('collaborators', collaborators);
+    this.props.setFormData(pendingValKey, null);
+  }
+
+  removeCollaborator(formElement, value) {
+    let collaborators = this.props.formData.collaborators || [];
+    collaborators = collaborators.filter(c => c.name !== value);
+    this.props.setFormData('collaborators', collaborators);
+  }
+
+  setCollaboratorEmail(formElement, value) {
+    let collabName = formElement.split('_')[1];
+    let collaborators = this.props.formData.collaborators;
+    let i = collaborators.findIndex(c => c.name === collabName);
+    collaborators[i].email = value;
+    this.props.setFormData('collaborators', collaborators);
+  }
+
+  toggleEmailNotify(e) {
+    if (e.target.name.indexOf('collabEmail') > -1) {
+      let collaborators = this.props.formData.collaborators;
+      let collabName = e.target.value;
+      let i = collaborators.findIndex(c => c.name === collabName);
+      collaborators[i].notify = !collaborators[i].notify;
+      this.props.setFormData('collaborators', collaborators);
+    } else if (e.target.name === 'leadInvestigatorEmail_notify') {
+      let currNotify = this.props.formData.notifyLead === undefined ? false : this.props.formData.notifyLead;
```

Comment:
I think `let currNotify = this.props.formData.notifyLead || false;` should work here.